localpdb/PDB.py

```diff
@@ -212,6 +212,18 @@ def _add_col_chains(self, data, added_col_name=[]):
                                           columns=added_col_name)
         self.__chains = pd.merge(self.chains, data, left_index=True, right_index=True, how='left')
 
+    def search_seq_motif(self, query, type_='prosite'):
```

Comment:
Add docstring with parameters and output format.

Reply:
Thank you, added